docs/source/index.md

```diff
@@ -94,5 +94,7 @@ getting_started/index
 concepts/index
 distributions/index
 contributing/index
+distribution_dev/index
```

Comment:
this should go away. I deleted and moved the content inside another section.

Reply:
done!